frontend/src/hooks/tripPlans.ts

```
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import type { TripPlan } from '../types/tripPlans.ts';
+
+export const fetchTripPlans = async (): Promise<TripPlan[]> => {
+    try {
+        const response = await axios.get<TripPlan[]>('http://localhost/api/trip-plan')
+        return response.data
+    } catch (error: any) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)